web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerDefault.java

```diff
@@ -0,0 +1,23 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+public class LinkCheckerDefault implements LinkCheckerInterface {
+    private String getCheckUrl(Element onlineResource) {
+        return LinkCheckerUtils.parseOnlineResource(onlineResource, LinkCheckerUtils.URL_XPATH);
+    }
+
+    @Override
+    public boolean check(Element onlineResource) {
+        String checkUrl = getCheckUrl(onlineResource);
+        if (checkUrl != null)
+            return LinkCheckerUtils.checkHttpUrl(checkUrl);
+
+        return true;
+    }
+
+    @Override
+    public String toString(Element onlineResource) {
```

Comment:
I was talking about `toString()` being duplicated.  Base class, maybe it would make sense to have:

```java
public abstract AbstractLinkChecker {
  public abstract boolean check(...);
  
  public String toString(...) {
    // magic happens
  }
}```

Pretty normal for an abstract class to have that in its name.